bin/stop-service.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+DIRNAME=`dirname $0`
+cd $DIRNAME/..
+
+if [ ! -f ./bin/shutdown.pid ]; then
+    echo "shutdown.pid not found!"
+else
```

Comment:
nice @wesleymonte. What if we do start-stop twice? the shutdown file keeps more than one pid?